docs/Toy/toy/dialect.py

```diff
@@ -150,6 +150,14 @@ def from_callable(name: str,
             Region.from_block_list([Block.from_callable(input_types, func)]),
             private=private)
 
+    @staticmethod
+    def from_callable_region(name: str,
+                             callable_region: tuple[Region, FunctionType],
+                             /,
```

Comment:
This looks at least weird, can you improve it? Is it possible?

Reply:
Do you mean the whitespace? I think the formatter treats it as a normal parameter, so it's on its own line. I kinda like it, makes it visible